release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1426,6 +1426,12 @@ In {product-title} v4.12, the LVM Operator has been renamed _Logical Volume Mana
 Now, you install it as a standalone Operator from the OpenShift Operator catalog.
 Logical Volume Manager Storage provides dynamic provisioning of block storage on a single, limited resources {sno} cluster.
 
+[discrete]
+[id="ocp-4-12-openstack-platform-support"]
+=== End of support for {rh-openstack} 16.1
+
+{product-title} no longer supports {rh-openstack} 16.1 as a deployment target. See [OpenShift Container Platform on Red Hat OpenStack Platform Support Matrix](https://access.redhat.com/articles/4679401) for complete details.
```

Comment:
I don't know that I like having this announce  mid 4.12's lifecycle (especially with it being a EUS release with a 24 month lifecycle); why wasn't this part the GA announcement?

Reply:
@sferich888 The request for this note came in well after release notes finalization for 4.12 GA. Not sure if Gil or @dulek can provide more background on it.

Reply:
This was hashed out by PM only by late January, we're just trying to correct a honest mistake of not having that decided before RNs were finalized. However the matrix [1] shows the correct version numbers for a while.

[1] https://access.redhat.com/articles/4679401